lua/nvim-lsp-ts-utils/client.lua

```diff
@@ -36,6 +39,36 @@ local edit_handler = function(_, _, workspace_edit)
     return { applied = status, failureReason = result }
 end
 
+local diagnostics_handler = function(err, method, result, client_id, buf_nr, config)
+    local forbidden_severities = o.get().filter_out_diagnostics_by_severity
```

Comment:
Could we rename these variables to match the names of the options? I had a moment of confusion testing this out.